README.md

```diff
@@ -42,10 +46,28 @@ Next, register the OpenTelemetry logback appender in `logback-spring.xml` (or `l
   </root>
 </configuration>
 ```
+                   
+## Log4j2
 
-todo: If you use log4j ...
-
-
+If you use log4j2, register the OpenTelemetry logback appender in `log4j2.xml`:
+              
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration packages="io.opentelemetry.instrumentation.log4j.appender.v2_17">
+  <Appenders>
+    <Console name="Console" target="SYSTEM_OUT">
+      <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
+    </Console>
+    <OpenTelemetry name="OpenTelemetryAppender"/>
+  </Appenders>
+  <Loggers>
+    <Root level="info">
+      <AppenderRef ref="OpenTelemetryAppender"/>
+      <AppenderRef ref="Console"/>
+    </Root>
+  </Loggers>
+</Configuration>
+```
 
 Finally, configure your application.yaml or application.properties either for Grafana Cloud or Grafana Agent.
 
```

Comment:
Can we move the `Configuration` header up and have `Grafana Cloud` and `Grafana Agent` as subsections?

### Configurations

#### Grafana Cloud
#### Grafana Agent

Reply:
Nice job with the changes!

Reply:
the installation header is supposed to cover everything for a typical installation.
moving config up would break that.

Reply:
I'm not those sections should be under Installation when the README states:

> Finally, **configure** your application.yaml or application.properties either for Grafana Cloud or Grafana Agent

Since we are mentioning configuring and properties, I think we should move `Grafana Cloud` and `Grafana Agent` under `Configuration`.

Reply:
found a way to have both - now "Configuration" is a subheading of "Getting Started" in #23